specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/virtualNetworkGateway.json

```diff
@@ -3621,6 +3634,91 @@
       ],
       "description": "An IPSec Policy configuration for a virtual network gateway connection."
     },
+    "VngClientConnectionConfiguration":
+    {
+      "properties": {
+        "vpnClientAddressPool": {
+          "$ref": "./virtualNetwork.json#/definitions/AddressSpace",
+          "description": "The reference to the address space resource which represents Address space for P2S VpnClient."
+        },
+        "virtualNetworkGatewayPolicyGroups":
+        {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
```

Comment:
should be ./network.json#/definitions/SubResource ?

Reply:
Please update the swagger otherwise the tooling check could not pass.

Reply:
Hi @lirenhe    I have finished the CI changes, it looks like all tests except the powershell sdk test is failing, not sure what happened, can you help take a look? thanks a lot!